.circleci/config.yml

```diff
@@ -35,7 +35,10 @@ commands:
           when: always
       - run:
           name: Upload JUnit XML Report
-          command: yarn junit:upload
+          command: |
+            curl -L "https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci_linux-x64" --output "/usr/bin/datadog-ci"
```

Comment:
Wouldn't using this instead of the `npm` package make this more difficult to test locally? Or do we not support running this locally in the first place?

Reply:
yeah it might not make much sense in this case.